db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171013213906) do
+ActiveRecord::Schema.define(version: 20171016172945) do
```

Comment:
You probably shouldn't commit this file.

Reply:
actually he has to because there's actual schema change.

Reply:
hah good point